### ISSUE TITLE
Google Doubleclick/DFP new tag format 

### ISSUE THREAD

**Issue by henrikpalm** [revive-adserver/revive-adserver#1012]
Add support for Doubleclick/DFP new tag-format.
Google has a "new" tag-format and i cant get them to work. Googles old legacy scripttags seems to work if i choose "Doubleclick/DFP" under "Generic HTML banner". If Revive wants to continue supporting Doubleclick then an update might be needed

Format of old tags that works:
<SCRIPT language='JavaScript1.1' SRC="https://ad.doubleclick.net/ddm/adj/123456789;sz=320x320;ord=[timestamp];dc_lat=;dc_rdid=;tag_for_child_directed_treatment=;tfua=?">
</SCRIPT>

Format of Doubleclick new tags below:
<ins class='dcmads' style='display:inline-block;width:320px;height:320px'
    data-dcm-placement='123456789'
    data-dcm-rendering-mode='script'
    data-dcm-https-only
    data-dcm-resettable-device-id=''
    data-dcm-click-tracker=''
    data-dcm-app-id=''>
  <script src='https://www.googletagservices.com/dcm/dcmads.js'></script>
</ins>

Please make sure that data-dcm-click-tracker='' is working also

**Comment by erikgeurts** [comment]
Some time ago, we gave up trying to keeping Google AdSense tag click tracking to work. It just happens that Google change things all the time, and it was impossible to keep up.

However, now with these new tags, the trick is actually rather simple:
- select "Do not alter HTML"
- in the data-dcm-click-tracker just add "{clickurl}"

I've worked with a few of my clients and tested this myself, and it works all the time.

In short: it's supported and in fact it always has been (in retrospect)

**Comment by henrikpalm** [comment]
Ok i believe that this is extremely simply for you to implement but i can understand that it sometimes is  hard to keep up with google. 
But how about letting the user create it's own adserving "recipies"!? 
It's almost always comes down to where to put the "{clickurl}" and if you supply the most common macros and a DB table it should be more flexible and futureproof. And then the user could combine that with a [featurerequest](https://forum.revive-adserver.com/topic/5195-html5-tags-custom-landingpage-urls/) that i for one would love to have a macro for

**Comment by erikgeurts** [comment]
In version 5.1.0, the functionality to support automatic click tracking on third party tags, including DFP/DCM/Google Ad Manager, has been removed. As such, this request can no longer be implemented (other than of course simply adding the {clickurl} macro manually). As such, this issue can now be closed.
